data-prepper-plugins/aws-lambda/src/main/java/org/opensearch/dataprepper/plugins/lambda/processor/AggregateResponseEventHandlingStrategy.java

```diff
@@ -5,13 +5,19 @@
 import org.opensearch.dataprepper.model.event.Event;
 import org.opensearch.dataprepper.model.record.Record;
 import org.opensearch.dataprepper.plugins.lambda.common.accumlator.Buffer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.util.List;
 
 public class AggregateResponseEventHandlingStrategy implements ResponseEventHandlingStrategy {
 
+    private static final Logger LOG = LoggerFactory.getLogger(AggregateResponseEventHandlingStrategy.class);
+
     @Override
-    public void handleEvents(List<Event> parsedEvents, List<Record<Event>> originalRecords, List<Record<Event>> resultRecords, Buffer flushedBuffer) {
+    public void handleEvents(List<Event> parsedEvents, List<Record<Event>> originalRecords,
+                             List<Record<Event>> resultRecords, Buffer flushedBuffer) {
```

Comment:
do we need to pass the `flushedBuffer` as a parameter? we are not using it here.